### PR TITLE
[4.0] Install from web: style of error change

### PR DESCRIPTION
**Pull request by rjharishabh**
Pull Request for Issue #33680  .

### Summary of Changes
Change error style according to Joomla 

####  Please disconnect  your computer from internet for few seconds

`Dashboard` > `System` > `Extensions under Install` > `Go to Install from Web` 
Apply PR
Do `npm i`
Repeat above steps and see the difference

### Actual result BEFORE applying this Pull Request
error-before

### Expected result AFTER applying this Pull Request
error-after

### Documentation Changes Required
No

### CODE REVIEW

File: plugins/installer/webinstaller/tmpl/default.php [hunk 1]
```diff
@@ -24,8 +24,20 @@
 			<h2 class="card-title"><?php echo Text::_('PLG_INSTALLER_WEBINSTALLER_INSTALL_WEB_LOADING'); ?></h2>
 		</div>
 	</div>
-	<div class="alert alert-error hidden" id="web-loader-error">
-		<a class="close" data-dismiss="alert">×</a><?php echo Text::_('PLG_INSTALLER_WEBINSTALLER_INSTALL_WEB_LOADING_ERROR'); ?>
+	<div class="hidden" id="web-loader-error">
+		<div id="system-message-container" aria-live="polite">
+			<joomla-alert type="danger" dismiss="true" class="joomla-alert--show" role="alert">
+				<button class="joomla-alert--close" aria-label="Close"><span aria-hidden="true">×</span></button>
```
Comment (dgrammatiko) [inline plugins/installer/webinstaller/tmpl/default.php]: Please remove this line

Comment (rjharishabh) [reply plugins/installer/webinstaller/tmpl/default.php]: Done 
Thanks @dgrammatiko

Comment (dgrammatiko) [reply plugins/installer/webinstaller/tmpl/default.php]: Actually, this is wrong. The message needs to be generated in the JS land. Here's what you need to do:

- add before the top PHP closing bracket `?>`: `Text::script('PLG_INSTALLER_WEBINSTALLER_INSTALL_WEB_LOADING_ERROR')`;
- remove everything but keep the empty `<div class="alert alert-error hidden" id="web-loader-error"></div>`
- in the js file add after https://github.com/joomla/joomla-cms/blob/5e08d610d3bdc7b6af252247f69d87d6c041cc6c/build/media_source/plg_installer_webinstaller/js/client.es6.js#L162

```js
Joomla.renderMessages({ danger: [Joomla.Text._('PLG_INSTALLER_WEBINSTALLER_INSTALL_WEB_LOADING_ERROR')] }, '#web-loader-error');
```

**Comment by drmenzelit** [comment]
@rjharishabh can you please check the comments from @dgrammatiko , I think you didn't make all the changes he was asking for

File: plugins/installer/webinstaller/tmpl/default.php (hunk 1 above)
Comment (dgrammatiko) [reply plugins/installer/webinstaller/tmpl/default.php]: Just FYI the wrong part here was `id="system-message-container"` The page already had this id so you can't reuse it, ids are unique

Comment (rjharishabh) [reply plugins/installer/webinstaller/tmpl/default.php]: Yes, that's the problem,
Now I made the changes suggested by you

**Comment by rjharishabh** [comment]
> @rjharishabh can you please check the comments from @dgrammatiko , I think you didn't make all the changes he was asking for

Done

**Comment by ChristineWk** [comment]
@rjharishabh Don't get error message

screen shot 2021-05-11 at 11 33 44
screen shot 2021-05-11 at 11 33 57

	Site 	File 	No Problems 	4.0.0-2021-05-07 	‎4.0.0-beta8-dev+pr.33743 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Comment by dgrammatiko** [comment]
@ChristineWk you misunderstood the testing instructions. Do not take the site offline but rather disconnect your computer from the internet...

**Comment by ChristineWk** [comment]
@dgrammatiko 

Aha :-) also no error message.
after 2nd attempt was then frozen screen 
Sorry about. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Comment by ChristineWk** [comment]
See:

screen shot 2021-05-11 at 12 17 12<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Comment by ghost** [comment]
I disconnected my computer from the internet but as i have the installation over launch.joomla.org i can't test ;-)

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on f8020b34ca49684b48c8aacf7d9b0267d1785828<br /><br />I am working on a Mac laptop and at the top of the screen there is a WiFi button with On/Off options. With the patch applied the error message is styled as described.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on f8020b34ca49684b48c8aacf7d9b0267d1785828<br /><br />After PR:

image

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33743">issues.joomla.org/tracker/joomla-cms/33743</a>.</sub>

**Pull request merged**